bitrise.yml

```diff
@@ -154,7 +154,7 @@ workflows:
 
   test_ndk_install:
     envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-samples/android-sdk21-tools-21.0.1.git
+    - SAMPLE_APP_URL: https://github.com/bitrise-samples/android-ndk-sample-project.git
```

Comment:
Can't recall why this exact repository is used for testing. If there is an option to keep this test, and have another separated test to run with the freshly wired in test case I think that might be better.

Reply:
The tests passed with the old repo, but only due to misconfiguration. The reason is the following:
[Configuration](https://developer.android.com/studio/projects/gradle-external-native-builds.html)
So when the ensure ran, it didn't found the NDK as a dependency, so it wasn't an issue, that the platforms were wrapped with an extra dir. This is the reason I changed the sample.